questionary/prompt.py

```diff
@@ -172,6 +174,20 @@ def unsafe_prompt(
                     "'when' needs to be function that accepts a dict argument"
                 )
 
+        # handle 'print' type
+        if _type == "print":
+            try:
+                message = _kwargs.pop("message")
+            except KeyError as e:
+                raise PromptParameterException("message") from e
+
+            # questions can take 'input' arg but print_formatted_text does not
+            # Remove 'input', if present, to avoid breaking during tests
+            _kwargs.pop("input", None)
+
+            print_formatted_text(message, **_kwargs)
+            continue
```

Comment:
I wonder if `answers[name] = None` if `name` is provided (rather than being a non-existent key in the answers dictionary)?

Reply:
I thought about including the print name in the answers dictionary as either:

1) setting the name to the print message
2) setting the name to `None` as you suggested

However, I think this is unnecessary and maybe even counterintuitive.  For one, it gives a different result whether or not `name` is provided. I don't think `name` should be required for a simple print.  Second, the python `print()` function returns `None` and I can't think of a case where I'd want to assign the return value of `print()` (which will always be `None`) to a variable.

I'm happy to change this behavior if you want me to but as a user of `questionary` it's not something I'd ever use.

Reply:
I'm not too keen on just ignoring the `name` argument, which is why I suggested returning `None`. However, you could instead throw an error if it is passed, stating that `name` is not supported for the `print` type. What do you think?

Reply:
I think it's cleaner to return `None` than throw an error as `name` is required for every other question type and someone who's just following the pattern of `{'name': name_value, 'type': type_value, ...}` might be initially puzzled about the error.  I'll modify the code to return None for name and add test.